### PR TITLE
8 patch

### PR DESCRIPTION
**Pull request by Sonia-Heer**

### CODE REVIEW

File: Models/models.js [hunk 1]
```diff
@@ -17,8 +17,7 @@ exports.fetchReviews = (sort_by = 'created_at', order = 'asc') => {
     if(!validOrderQueries.includes(order)){
         return Promise.reject({status: 400, msg: 'invalid order query' })
```
Comment (pippaamy) [inline Models/models.js:18]: put this file in your git ignore so its not being pushed up!!!!

File: Models/models.js [hunk 2]
```diff
@@ -59,3 +58,21 @@ exports.fetchReviewIdComments = (review_id) => {
     });
 };
 
+exports.updatedReview = (review_id, inc_votes) => {
+    if(inc_votes !== undefined){
+        return checkReviewExists(review_id)
+        .then((exists) => {
```
Comment (pippaamy) [inline Models/models.js:64]: would be more readable if you had (inc_votes === undefined)  --> promise.reject etc

**Review by pippaamy — COMMENTED** [review]
good job, your model is a bit tricky to read! I've popped a comment in there to have a think about!

File: Models/models.js [hunk 3]
```diff
@@ -59,7 +59,9 @@ exports.fetchReviewIdComments = (review_id) => {
 };
 
 exports.updatedReview = (review_id, inc_votes) => {
-    if(inc_votes !== undefined){
+    if(inc_votes === undefined){
+        return Promise.reject({ status: 400, msg: "bad request" })
+    }else{
         return checkReviewExists(review_id)
```
Comment (robert112233) [inline Models/models.js:65]: Can you explain the thought process behind checking if the review exists? If there was no review found at that WHERE clause SQL would return empty rows, you could use conditional logic on these empty rows to throw a 404 ? I might be wrong though. 

2 db queries are slightly less efficient that's all

File: Models/models.js [hunk 4]
```diff
@@ -71,8 +73,6 @@ exports.updatedReview = (review_id, inc_votes) => {
             return Promise.reject({ status: 404, msg: "Not found"})
```
Comment (robert112233) [inline Models/models.js:73]: Line 70 you could have an if statement which either returns results.rows[0] or Promise.reject

**Review by robert112233 — CHANGES_REQUESTED** [review]
I think you can remove the checkIfReviewExists Lmk if you disagree

**Pull request merged**